server/paymentProviders/paypal/payment.ts

```diff
@@ -78,10 +80,6 @@ const recordTransaction = async (
     },
   };
 
-  if (createdAt) {
-    transactionData['createdAt'] = createdAt;
-  }
-
```

Comment:
Removing this introduced seems to have introduced a breaking change: https://github.com/opencollective/opencollective/issues/7296